tools/helper/src/node/page/excerpt.ts

```diff
@@ -0,0 +1,253 @@
+// eslint-disable-next-line vue/prefer-import-from-vue
+import { isHTMLTag, isMathMLTag, isSVGTag } from '@vue/shared'
+import type { AnyNode, Element } from 'cheerio'
+import { load } from 'cheerio'
+import matter from 'gray-matter'
+import type { App, Page } from 'vuepress/core'
+import { isLinkHttp, removeEndingSlash } from 'vuepress/shared'
+import { isAbsoluteUrl, isArray } from '../../shared/index.js'
+
+const HEADING_TAGS = ['h1', 'h2', 'h3', 'h4', 'h5', 'h6']
+
+export interface PageExcerptOptions {
+  /**
+   * Excerpt separator
+   *
+   * 摘要分隔符
+   *
+   * @default "<!-- more -->"
+   */
+  excerptSeparator?: string
```

Comment:
I think just `separator` is fine since its part of the `PageExcerptOptions`.

I was trying to thing if there was a better word then separator but the best I could think of was `boundary` and I don't know that is really any better.

Reply:
separator is using at gray-matter, so I think it's good to inherit this option name.